internal/replacer/replacer_test.go

```diff
@@ -1,4 +1,4 @@
-package replacer
+package replacer_test
```

Comment:
Without this, there are cyclic dependencies in the shell package.
`shell` -> `olfactor` -> `replacer` -> `redact` -> `shell`

But the `replacer` only needs the redactor for tests, so making a separate `_test` package fixed this.